### PR TITLE
Update missing types from rc-slider

### PR DESCRIPTION
**Pull request by paustint**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/react-component/slider
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

react-component_slider__react_slider

### CODE REVIEW

**Comment by mantasmarcinkus** [comment]
👍

**Comment by uwolfer** [comment]
I have done the same here: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/20868, but your PR contains more changes. So I'll close my one. I don't know how this project handles it, but I'd prefer to squash your 3 commits into one because they are only follow-up fixes.

File: types/rc-slider/index.d.ts [hunk 1]
```diff
@@ -1,6 +1,8 @@
 // Type definitions for rc-slider 8.1
```
Comment (uwolfer) [inline types/rc-slider/index.d.ts]: `dotStyle` and `activeDotStyle` are included in 8.2 only (not in 8.1).

File: types/rc-slider/index.d.ts [hunk 2]
```diff
@@ -103,6 +105,16 @@ export interface CommonApiProps {
      * The style used for the track base color.
      */
     railStyle?: React.CSSProperties;
+
+    /**
+     * The style used for the track base color.
```
Comment (uwolfer) [inline types/rc-slider/index.d.ts]: This text is wrong.

File: types/rc-slider/index.d.ts [hunk 3]
```diff
@@ -103,6 +105,16 @@ export interface CommonApiProps {
...
+     */
+    dotStyle?: React.CSSProperties;
+
+    /**
+     * The style used for the track base color.
```
Comment (uwolfer) [inline types/rc-slider/index.d.ts]: This also.

**Review by uwolfer — CHANGES_REQUESTED** [review]

**Comment by paustint** [comment]
@uwolfer - thanks for the review. I had some issues because I had an old version of the repo and someone made a change so I had to start over and in my haste I missed updating the description text. I attempted to squash my commits but had already pushed to GH so this was not possible.  Would it be acceptable to squash on merge from whomever merges, or should I open up a new PR?

**Comment by uwolfer** [comment]
@paustint: You can squash your commits and then do a push --force into your remote branch. This will update the PR.

File: types/rc-slider/index.d.ts (hunk 3 above)
Comment (paustint) [reply types/rc-slider/index.d.ts]: Corrected in 939d165

File: types/rc-slider/index.d.ts (hunk 2 above)
Comment (paustint) [reply types/rc-slider/index.d.ts]: Corrected in 939d165

File: types/rc-slider/index.d.ts (hunk 1 above)
Comment (paustint) [reply types/rc-slider/index.d.ts]: Corrected in 939d165

**Review by uwolfer — APPROVED** [review]

**Pull request merged**